modules/manage/partials/iceberg/query-iceberg-topics.adoc

```diff
@@ -43,7 +43,7 @@ endif::[]
 {"user_id": 2324, "event_type": "BUTTON_CLICK", "ts": "2024-11-25T20:23:59.380Z"}
 ----
 
-=== Topic with schema (`value_schema_id_prefix` mode)
+=== Topic with schema (`value_schema_id_prefix` or `value_subject_latest` mode)
```

Comment:
Do we need a separate section for value_subject_latest? See https://deploy-preview-1068--redpanda-docs-preview.netlify.app/current/manage/iceberg/query-iceberg-topics/#topic-with-schema-value_schema_id_prefix-or-value_subject_latest-mode